setup.py

```diff
@@ -80,7 +80,6 @@ def read(*rnames):
         ]
     ),
     setup_requires=[
-        'eggtestinfo',
         'zope.testrunner',
```

Comment:
As @icemac suggested in https://github.com/zopefoundation/zope.app.publication/pull/14, let's not setup_requires zope.testrunner.